clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperations.groovy

```diff
@@ -67,4 +67,5 @@ final class AtomicOperations {
 
   // Serialize operations
   public static final String SERIALIZE_APPLICATION = "serializeApplication"
```

Comment:
This line isn't touched in this pr, but hadn't we discussed changing `serializeApplication` to `captureSnapshot` or something like that?